docs/deployment.md

```diff
@@ -98,7 +98,8 @@ FROM node:alpine AS builder
 WORKDIR /app
 COPY . .
 COPY --from=deps /app/node_modules ./node_modules
-RUN yarn build
+RUN yarn build \
+  yarn install --production --ignore-scripts --prefer-offline
```

Comment:
Don't we really just need `--production` here? Why the others? Can we also keep on one line?

Reply:
I added the additional flags because I followed the recommendation in this GitHub issue:
https://github.com/yarnpkg/yarn/issues/696#issuecomment-258418656

Here's the comment for convenience:
> yarn install --production is close, but it does a few things prune doesn't. It's a little annoying to have to build native modules twice in a CI pipeline just to prune. You can prevent rebuilding packages that already exist in cache with the --ignore-scripts flag. Additionally, prune doesn't check npm registry, so add --prefer-offline.
>
> So, yarn install --production --ignore-scripts --prefer-offline == npm prune --productio